semantic_model_generator/generate_model.py

```diff
@@ -97,6 +98,11 @@ def _raw_table_to_semantic_context_table(
                     description=_PLACEHOLDER_COMMENT,
                 )
             )
+        elif col.column_type in OBJECT_DATATYPES:
```

Comment:
do we need col.column_type.upper()?

Reply:
No as the connector pulls in caps. Will add though.